src/Kernel/client/kernel.ts

```diff
@@ -65,7 +65,16 @@ class Kernel {
                 if (stateDivId != null) {
                     let stateDiv = document.getElementById(stateDivId);
                     if (stateDiv != null) {
-                        let { chart: chart } = createNewCanvas(stateDiv);
+                        // Create necessary elements so a large chart will scroll
+                        let chartWrapperDiv = document.createElement("div");
+                        chartWrapperDiv.style.overflowX = "scroll";
+                        let innerChartWrapperDiv = document.createElement("div");
+                        innerChartWrapperDiv.style.height = "350px";
+                        innerChartWrapperDiv.id = "innerChartWrapper";
```

Comment:
There may in general be more than one such `<div>` element, such that assigning the same ID to each could cause problems later. Would it be possible to either pass the `HTMLElement` object directly to `fitChart`, or to generate a unique ID for each chart wrapper div?

Reply:
ah good point thank you. Fixed it so it refers to this element through `chart.canvas.parentElement` instead